### PR TITLE
Fix datacoord panics when collection info not found

### PR DESCRIPTION
**Pull request by congqixia**
Fix #19705 

/kind bug

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: internal/datacoord/util.go [hunk 1]
```diff
@@ -118,7 +118,11 @@ func FilterInIndexedSegments(meta *meta, indexCoord types.IndexCoord, segments .
 		collectionSegments[collectionID] = append(collectionSegments[collectionID], segment.GetID())
 	}
 	for collection := range collectionSegments {
-		schema := meta.GetCollection(collection).Schema
+		collInfo := meta.GetCollection(collection)
+		if collInfo == nil {
+			continue
```
Comment (jaime0815) [inline internal/datacoord/util.go:123]: Maybe adding a log will be better.

Comment (congqixia) [reply internal/datacoord/util.go:123]: On it

**Pull request merged**